guides/getting-started/set-up/anoncreds-rs.md

```diff
@@ -0,0 +1,126 @@
+# Anoncreds-rs
+
+[AnonCreds](https://github.com/hyperledger/anoncreds) is a direct
+implementation of the [AnonCreds V1.0
+specification](https://hyperledger.github.io/anoncreds-spec/) that provides
+functionality like; creating a schema object, creating a credential definition
+object, creating a credential, verifying a proof presentation and much more.
+
+:::caution
+
+Support for AnonCreds in Aries Framework JavaScript is currently experimental.
+We recommend new projects to use AnonCreds from the start, and also to migrate
+existing projects to Anoncreds, from the indy-sdk. However, projects may
+experience some issues. If you encounter any issues, please [open an
+issue](https://github.com/hyperledger/aries-framework-javascript/issues/new).
+
+Currently, there are few limitations to using AnonCreds.
+
+- React Native projects leveraging [Expo](https://expo.dev) are not supported
+  at the moment. Support for Expo will be added soon.
+- When running in Node.JS, only Node.JS 18 is suported for now. See [Supported
+  Node.JS versions for AnonCreds](#supported-nodejs-versions-for-anoncreds)
+
+:::
+
+### Installing AnonCreds
```

Comment:
This guide is for setting up AnonCreds RS right? How does it relate to setting up AnonCreds in general? Is that handled by the specific tutorials, or do we also need to update the indy sdk tutorial to add the anoncreds module?

Reply:
Yes, I would say just anoncreds-rs. I think if we want to add some context surrouding the anoncreds package, we should do it as a `concept` and this would just be a minimal setup for an agent using the anoncredsrs library.